src/actions/slack/legacy_slack/slack.ts

```diff
@@ -28,9 +29,11 @@ https://github.com/looker/actions/blob/master/src/actions/slack/legacy_slack/REA
 
   async form(request: Hub.ActionRequest) {
     const form = new Hub.ActionForm()
+    const channelType = _.isNil(request.formParams.channelType) && request.formParams.channelType === "users"
```

Comment:
it should be `||` i think, _.isNil(request.formParams.channelType) || request.formParams.channelType === users